scripts/deploy.bash

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Deploy assets to DEPLOY_HOST.
+
+# Required environment variables:
```

Comment:
you could check the environment variables with 
[[ -z "$ENV" ]]

Reply:
will do.